java/serving/src/test/resources/docker-compose/feast10/definitions.py

```diff
@@ -7,7 +7,7 @@
 from feast.feature_view import FeatureView
 from feast.entity import Entity
 from feast.feature_service import FeatureService
-from feast.on_demand_feature_view import RequestDataSource, on_demand_feature_view
+from feast.data_source import RequestSource, on_demand_feature_view
```

Comment:
`on_demand_feature_view` should come from `feast.on_demand_feature_view`, `RequestSource` from `feast.data_source`.

Reply:
Oh srry, I just fixed that.